lib/routes/avatar/get.js

```diff
@@ -33,13 +34,19 @@ module.exports = {
     }
   },
   handler: function avatar(req, reply) {
-    db.getSelectedAvatar(req.auth.credentials.user)
+    var uid = req.auth.credentials.user;
+    db.getSelectedAvatar(uid)
       .then(avatarOrEmpty)
       .done(function (result) {
         var rep = reply(result);
         if (result.id) {
           rep = rep.etag(result.id);
         }
+        var info = {
+          event: 'avatar.get',
+          uid: uid
+        };
+        logger.info('activityEvent', info);
```

Comment:
> @vladikoff is it better to measure avatar-reads or avatar-writes for our metrics purposes?

I think avatar-write should be more useful, but avatar-read also has a use case:
with 
avatar-write:
"Users who uploaded an avatar and still using" vs "Users who did not and still using"

avatar-get:
"User fetching avatar after X weeks"